Gemfile

```diff
@@ -35,7 +35,7 @@ group :development do
   gem 'capistrano-passenger', require: false
   gem 'capistrano-rails', require: false
   gem 'dlss-capistrano', require: false
-  gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'listen'
```

Comment:
Typically we want to leave a version spec in Gemfile if possible.  In this case '~> 3.7' would be a good choice.  This allows the minor version to float up but it won't automatically upgrade to 4.0 without changing the version spec.